src/oidc-keychain/oidc-keychain

```diff
@@ -10,7 +10,7 @@ usage()
 {
   (
   echo "Usage: $ME [-?Vk] [--help|--usage|--version] [--kill] \\"
-  echo "           [oidc-agent options] [ACCOUNT_SHORTNAME...]"
+  echo "           [oidc-agent options] [--acounts ACCOUNT_SHORTNAME...]"
```

Comment:
Typo here: --acounts should be --accounts

Reply:
Right